index.html

```diff
@@ -212,98 +212,84 @@ <h2 class="about-header">About myself</h2>
-            <li>
-              <a href="#"
-                ><img
-                  class="btn"
-                  src="assets/icons/angellist.png"
-                  alt="Angellist icon"
-              /></a>
-            </li>
-          </ul>
+        <div class="div-connect2">
+          <h4 class="let-connect">LET’S CONNECT</h4>
+          <div class="social-media-icons">
+            <ul>
+              <li>
+                <a href="#"
+                  ><img
+                    class="btn"
+                    src="assets/icons/twitter.png"
+                    alt="Twitter icon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    class="btn"
+                    src="assets/icons/Linkedin icon.png"
+                    alt="LinkedIn icon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    class="btn"
+                    src="assets/icons/medium.png"
+                    alt="Medium icon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    class="btn"
+                    src="assets/icons/github.png"
+                    alt="Github icon"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    class="btn"
+                    src="assets/icons/angellist.png"
+                    alt="Angellist icon"
+                /></a>
+              </li>
+            </ul>
+          </div>
+          <button class="button-get-resume">Get my resume</button>
         </div>
-        <button class="button-get-resume">Get my resume</button>
       </div>
       <div class="languages">
-        <ul>
-          <li class="li-categories">
-            <div>
+            <div class="languages-title">
               <h4>Languages</h4>
               <img src="assets/icons/Expand_Arrow_16px.png" alt="" />
             </div>
-            <ul>
+            <ul class="ul-languages">
               <li class="languages-items-li">
-                <div class="languages-items">
                   <img src="assets/images/js.png" alt="" />
                   <p>JavaScript</p>
-                </div>
               </li>
               <li class="languages-items-li">
-                <div class="languages-items">
                   <img src="assets/images/html.png" alt="" />
                   <p>HTML</p>
-                </div>
               </li>
               <li class="languages-items-li">
-                <div class="languages-items">
                   <img src="assets/images/css.png" alt="" />
                   <p>CSS</p>
-                </div>
               </li>
             </ul>
           </li>
-          <li class="li-categories">
-            <div>
+          <div class="languages-module">
```

Comment:
[OPTIONAL]
- [ ] Kindly remove that extra closing `li` tag on line `283` for cleaner code.

Reply:
Thanks a lot dear @Ofarouq310